tiledb/core.cc

```diff
@@ -988,6 +985,17 @@ class PyQuery {
       buf.data_vals_read += data_vals_num;
       buf.offsets_read += offset_elem_num;
       buf.validity_vals_read += validity_elem_num;
+
+      if ((ssize_t)(buf.data_vals_read * buf.elem_nbytes) >
```

Comment:
Use `Py_ssize_t` instead of `ssize_t`. This should fix the build errors on Windows, Python 3.10.